rmarkdowns/Week_3.Rmd

```diff
@@ -90,6 +90,10 @@ available_tags("healthcare")
 The first step in creating an `osmdata` query is defining the geographical area we want to include in the query. This can be done by defining a bounding box that defines a geographical area by its bounding latitudes and longitudes. The osmdata package provides a function `getbb()` to retrieve the bounding box of a place using its name. We can now create the bounding box of Maui island in Hawaiʻi.
 
 **Challenge 2: Create a bounding box for your project target area.**
+```{r}
+bb <- getbb("damitta")
```

Comment:
Hi Amr, it looks like OSM uses the spelling "damietta". Try changing the spelling and see if that works better for the rest of the assignment! (You may have to choose a larger area, but try Damietta first)

Reply:
Actually, I just ran the schools chunks, and it looks like the OSM data only has one school in that area. You can continue and map just the one school, or you can look at a larger area. Up to you!